packages/block-library/src/heading/variations.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import HeadingLevelIcon from './heading-level-icon';
+
+const variations = [ 1, 2, 3, 4, 5, 6 ].map( ( level ) => ( {
+	name: `heading-${ level }`,
+	/* translators: %d: heading level. */
+	title: sprintf( 'Heading %d', level ),
+	icon: <HeadingLevelIcon level={ level } />,
+	attributes: { level },
+	scope: [ 'inserter' ],
```

Comment:
Do we want to include `isActive` to show different titles and icons in the block editor UI?

Reply:
Yeah, this can be discussed as well. I intentionally left it out for start, as I thought it was not so much needed -but of course that was a personal opinion 😄